From 886b9afcc566dd35bdea8a93f4c4dd5b4a2cc14f Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Mon, 24 Sep 2012 13:12:42 +0200
Subject: [PATCH] systemd/initrd-switch-root.target: run switch-root.target
 after cryptsetup

Ensure, that all crypto devices have been queried.
---
 modules.d/98systemd/initrd-switch-root.target | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/98systemd/initrd-switch-root.target b/modules.d/98systemd/initrd-switch-root.target
index 74647a5..05e0117 100644
--- a/modules.d/98systemd/initrd-switch-root.target
+++ b/modules.d/98systemd/initrd-switch-root.target
@@ -12,6 +12,7 @@ Description=Switch Root
 DefaultDependencies=no
 Requires=initrd-switch-root.service
 Before=initrd-switch-root.service
+After=cryptsetup.target
 AllowIsolate=yes
 Wants=systemd-journald.service
 ConditionPathExists=/etc/initrd-release
